proto/v1alpha1/rode.proto

```diff
@@ -115,8 +115,22 @@ service Rode {
   }
 }
 
+enum ResourceType {
+  UNSPECIFIED = 0;
```

Comment:
```suggestion
  RESOURCE_TYPE_UNSPECIFIED = 0;
```

I couldn't find a good source for this, but the enum values are scoped to the package and not the enum itself. So it's not possible to reuse the same enum value name. It seems like the convention is to [prefix with the enum name ](https://docs.buf.build/lint-rules/#enum_value_prefix) (or at least on the entries that are likely to clash with something else). 

I tested this out as well, by adding another enum:
```proto
enum OtherType {
  UNSPECIFIED = 0;
  FOO = 1;
}
```

and got this output:
>proto/v1alpha1/rode.proto:132:3: "UNSPECIFIED" is already defined in "rode.v1alpha1".
proto/v1alpha1/rode.proto:132:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "UNSPECIFIED" must be unique within "rode.v1alpha1", not just within "OtherType".

This may not ever be a problem, but it might be good to get that convention in place (and maybe even use something like `buf` for checking the protos).

Reply:
good catch, I can change this